barista-compose/src/main/java/com/alorma/barista_compose/assertion/internal/matcher/DrawableMatcher.kt

```diff
@@ -0,0 +1,23 @@
+package com.alorma.barista_compose.assertion.internal.matcher
+
+import androidx.annotation.DrawableRes
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.SemanticsPropertyKey
+import androidx.compose.ui.semantics.SemanticsPropertyReceiver
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.test.SemanticsMatcher
+import androidx.compose.ui.test.junit4.ComposeTestRule
+
+fun Modifier.setDrawableSemantic(@DrawableRes id: Int): Modifier {
```

Comment:
The idea is to set this modifier on production code?

Reply:
uhmm, then the code for the modifier shouldn't go on the same module than barista, as barista is only used for tests (`andoridTestimplementation`)

Reply:
maybe we need ... "barista-helpers"  or something

Reply:
OK! I had doubts because I've had to load this module in the project to run the test. So, you propose to create a new one called barista-helpers and move there the DrawableMatcher file, don't you?